pyvista/plotting/charts.py

```diff
@@ -1031,6 +1031,9 @@ def __init__(self, size=(1, 1), loc=(0, 0)):
         if loc is not None:
             self.loc = loc
 
+    def deep_clean(self):
+        """Clean up the chart."""
+
```

Comment:
Should we raise `NotImplementedError` here?  Otherwise, classes that inherit this will end up with methods that claim to do something, but do nothing.

Reply:
Then I need to try-except in one of the calling methods, as this `deep_clean` method is called for all chart objects right now (not just the ChartMPL's which have something to clean up). Personally I don't think it's bad these methods do nothing in child classes if there's nothing significant to clean up?

Reply:
Resolved/removed in #2031.